reg_tests/test_files/ablNeutralStat/README

```diff
@@ -0,0 +1,73 @@
+Neutral Atmospheric Boundary Layer Case
+
+08 June 2020
+
+This case is a variant of ablNeutralEdge.  It usese the same case, but
+runs a post-processing script (passfail.py) at the end to check with
+ABL statistics averaging is correct.
+
+**From ablNeutralEdge:**
+This case is an example for running a large-eddy simulation of a neutral-
+stability atmospheric boundary layer.  The mesh resolution is much coarser 
+than would be run in practice, but that can be easily changed for a real 
+case.  The case should be run for at least 16,000 s to acheive a level
+of quasi equlibrium.  We recommend running this case with 10 m uniform
+resolution in all directions (it is set up for uniform 40 m resolution
+for this regression test).  The stability is neutral with no heat flux
+at the ground, but the boundary layer is capped with a stable inversion
+layer.  Because of constant density used here, because we do not include
+pressure work term in the enthalpy equation, and becuase we use Boussinesq
+buoyancy, temperature is potential temperature.
+
+- To build the mesh, use NaluWindUtils abl_mesh with nalu_abl_mesh.yaml
```

Comment:
`NaluWindUtils` was renamed to `wind-utils` when we imported it into `exawind` org.

Reply:
Changed the name in the new README.